connectors/src/lib/models/microsoft.ts

```diff
@@ -170,7 +170,7 @@ MicrosoftNodeModel.init(
       allowNull: false,
     },
     internalId: {
-      type: DataTypes.STRING,
```

Comment:
why add a limit there? not sure it's done elsewhere
but then, why not? i'd go for 1024 though just to be sure